internal/connect/api.go

```diff
@@ -195,6 +233,14 @@ func makeSysInfoBody(distroTarget, namespace string, instanceData []byte) ([]byt
 	payload.InstanceData = string(instanceData)
 	payload.Namespace = namespace
 
+	uptimeLog, err := readUptimeLogFile(UptimeLogFilePath)
+	if err != nil {
```

Comment:
Yes, we can take the pessimistic approach. Should we also introduce logging facility here to at least log the errors that we are ignoring to aid troubleshooting?

Reply:
Since we are going with an option to explicitly enable the feature. I think we should error out if we can't collect system uptime logs. At the very least, we should find a way to log the errors to help troubleshooting.

Reply:
Logging is better as we are adding this information to an api request payload. The UX will suffer greatly if customers can't perform operations.

Reply:
Will log it at `Info` level.